### PR TITLE
Add additional section to Job cli tutorial

### PR DESCRIPTION
**Pull request by chesterxgchen**
### Description
Since we added the site-specific configuration section, we update the tutorials to include this. This PR includes 

* Advanced section on the Job CLI 
* Fix typos and comments based on the previous comments
* Re-write the configure file look up section and removed file-based app looked up based on @IsaacYangSLA's feedback. 
* Fix a bug ( removed extra configure files) 

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/tutorials/README.md [hunk 1]
```diff
@@ -12,3 +12,5 @@ Please make sure you set up a virtual environment and install and configure Jupy
   * Goes through the different commands of the FLARE API.
 * [NVFLARE in POC Mode](./setup_poc.ipynb)
   * Shows how to use POC mode.
+* [Job CLI](./job_cli.ipynb)
+  * Shows how to use JOB cli commands
```
Comment (YuanTingHsieh) [inline examples/tutorials/README.md:16]: minor change suggest: Job CLI

File: job_templates/sag_pt/config_fed_client.conf [hunk 2]
```diff
@@ -55,10 +55,10 @@
     }
   ],
 
-  # this defined an array of task data filters. If provided, it will control the data to server
+  # this defined an array of task data filters. If provided, it will control the data from server controller to client executor
   task_data_filters =  []
 
-  # this defined an array of task result filters. If provided, it will control the result to server
+    # this defined an array of task result filters. If provided, it will control the result from client executor to server controller
   task_result_filters = []
```
Comment (YuanTingHsieh) [inline job_templates/sag_pt/config_fed_client.conf:62]: To add more to this, we also have client side controller,
What is a more accurate way to describe this?
Should we explain the "direction" here (IN/OUT/INOUT)?
Or we should not expose too many details?

What do you think @yanchengnv @yhwen ?

File: nvflare/tool/job/config/configer.py [hunk 3]
```diff
@@ -226,56 +228,45 @@ def parse_cli_config(cli_configs: List[str], app_names: List[str], job_folder) -
     return app_cli_config_dict
 
 
-def build_config_file_indices(job_folder: str) -> Dict[str, Dict[str, Tuple]]:
-    excluded = ["info"]
-    included = ["config_fed_client", "config_fed_server", "meta"]
+def build_config_file_indices(job_folder: str, app_names: List[str]) -> Dict[str, Dict[str, Tuple]]:
+    config_included = ["config_fed_client", "config_fed_server"]
```
Comment (YuanTingHsieh) [inline nvflare/tool/job/config/configer.py:232]: For these string constants, should we put them in a share place?

I think we have somewhere else also using these strings as well. (Also "app", "config")?

File: job_templates/sag_pt_deploy_map/site-1/config_fed_client.conf [hunk 4]
```diff
@@ -55,10 +55,10 @@
     }
   ],
 
-  # this defined an array of task data filters. If provided, it will control the data to server
+  # this defined an array of task data filters. If provided, it will control the data from server controller to client executor
   task_data_filters =  []
 
-  # this defined an array of task result filters. If provided, it will control the result to server
+    # this defined an array of task result filters. If provided, it will control the result from client executor to server controller
```
Comment (IsaacYangSLA) [inline job_templates/sag_pt_deploy_map/site-1/config_fed_client.conf:61]: No need to change.  But I just noticed that there is one extra space at the beginning of this line.

**Review by IsaacYangSLA — APPROVED** [review]
Thanks.  LGTM.

**Review by IsaacYangSLA — APPROVED** [review]
Approve again after PR got new commits.

**Review by YuanTingHsieh — DISMISSED** [review]
Please fix unit test. Changes LGTM

**Review by YuanTingHsieh — APPROVED** [review]

**Pull request merged**